internal/signalfx-agent/pkg/monitors/prometheusexporter/conversion.go

```diff
@@ -96,27 +138,51 @@ func makeHistogramDatapoints(name string, ms []*dto.Metric) []*datapoint.Datapoi
 
 		//nolint:protogetter
 		if h.SampleCount != nil {
-			dps = append(dps, sfxclient.Cumulative(name+"_count", dims, int64(h.GetSampleCount()))) //nolint:gosec
+			metric := pmetric.NewMetric()
+			metric.SetName(name + "_count")
+			sum := metric.SetEmptySum()
+			sum.SetIsMonotonic(true)
+			sum.SetAggregationTemporality(pmetric.AggregationTemporalityCumulative)
+			dp := sum.DataPoints().AppendEmpty()
+			dp.SetIntValue(int64(h.GetSampleCount())) //nolint:gosec // disable G115
+			_ = dp.Attributes().FromRaw(dims)
+			dps = append(dps, metric)
```

Comment:
So, we are creating one metric with one data point for each Prometheus time series. Is that correct? I don't see any combining logic down the road...

Reply:
Looks like the existing `sfxDatapointsToPDataMetrics` implementation does pretty much the same though...

Reply:
Yes, this is a simple follow through to make sure we get the same behavior for now.